src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -18,12 +21,19 @@ public class RacingCarController {
 
```

Comment:
`@Service`, `@Repository`를 활용해주셨네요.
스프링의 의존성 주입을 활용해보시면 좋을듯합니다!

참고자료 : [[Spring Boot] 스프링 빈(bean)이란? 스프링 빈 등록하는 방법](https://velog.io/@falling_star3/Spring-Boot-%EC%8A%A4%ED%94%84%EB%A7%81-%EB%B9%88bean%EA%B3%BC-%EC%9D%98%EC%A1%B4%EA%B4%80%EA%B3%84)


`ex)`
```suggestion
    private final RacingCar racingCar;

    public RacingCarController(RacingCar racingCar) {
        this.racingCar = racingCar;
    }
```